lib/__tests__/fixtures/style-tag.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">
+<title>Document</title>
+<style type="text/sugarss">
+/* Cannot trigger a syntax warning */
+a
```

Comment:
Change comment here and in other fixtures: "should trigger warning ("block-no-empty") but is parsable - should not trigger syntax error"